Sources/BraveWallet/Settings/Web3SettingsView.swift

```diff
@@ -171,6 +171,17 @@ private struct WalletSettingsView: View {
   }
 
   var body: some View {
+    if !keyringStore.isDefaultKeyringCreated {
```

Comment:
maybe reverse this condition, easier to read :D

Reply:
Thanks! Addressed in [fb3d6ac](https://github.com/brave/brave-ios/pull/7313/commits/fb3d6ac5439f566883c0bbfa176492266ac6b3d0)